WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CardPresentPaymentsOnboardingPresenter.swift

```diff
@@ -79,6 +79,6 @@ final class CardPresentPaymentsOnboardingPresenter: CardPresentPaymentsOnboardin
     }
 
     func refresh() {
-        onboardingUseCase.forceRefresh()
+        onboardingUseCase.refreshIfNecessary()
```

Comment:
as previous note about "race condition which may not be a race condition any more"...

Reply:
Same as before.